### DISCUSSION TITLE
permanent password in Ubuntu 23.10 not working

### DISCUSSION THREAD

**Discussion by Pojay-hcz** [rustdesk/rustdesk#6604] [General]
### Bug Description

I have set a permanent password (replacing the OTP feature). Access from another Rustdesk Client from within my LAN always results in a "wrong password" error. Randomly a window opens on the target machine which only shows a "cancel" button but no "accept" button.
Usage of OTP works flawless.
I would appreciate if someone could guide me into the right direction. THX

### How to Reproduce

this behaviour is reproducible. I tried to access the target from 2 different machines in my LAN (from MacOS and iOS)

### Expected Behavior

that a permanent password (which complies with the password rules of Rustdesk) is accepted from the target machine

### Operating system(s) on local side and remote side

Ubuntu 23.10 on Raspberry Pi(local) -> OSX 14.1.2 / iPadOS 17.1.2 (remote)

### RustDesk Version(s) on local side and remote side

1.2.3-aarch64.deb -> 1.1.9 (OSX) / 1.2.2 (iPadOS)

### Screenshots

<img width="633" alt="Bildschirmfoto 2023-12-03 um 17 00 26" src="https://github.com/rustdesk/rustdesk/assets/48587984/605f8c24-a61d-4d2d-a623-bc5817bccfe4">

**Comment by 21pages** [comment]
Check whether `Accept sessions via password` and `Use one-time password` are selected by mistake on the controlled side.

![1701599009591](https://github.com/rustdesk/rustdesk/assets/14891774/b0653447-679c-4c2a-b789-3899a7ed4012)
